pylsp/lsp_types.py

```diff
@@ -0,0 +1,61 @@
+from typing import TypedDict, Optional, Union, List, NewType
```

Comment:
I created this file as faster type look up. The alternative was to click through the LSP sepcs, which is doable but slower, also because there is more visual clutter.